pages/api/headerList.ts

```diff
@@ -0,0 +1,22 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const handler = (req: NextApiRequest, res: NextApiResponse) => {
+  const headerList = [
+    { title: 'Главная', href: '/' },
+    { title: 'Услуги', href: '/services' },
+    { title: 'Косметика', href: '/cosmetics' },
+    { title: 'Блог', href: '/blog' },
+    { title: 'Франшиза', href: '/franchising' },
+    { title: 'Инвестиции', href: '/investment' },
+    { title: 'Вакансии', href: '/vacancies' },
+    { title: 'Барберы', href: '/barbers' },
+    { title: 'Контакты', href: '/contacts' },
+    { title: 'Филиалы', href: '#' },
+  ];
+
+  res.status(200).json({
+    success: true,
+    headerMainList: headerList,
```

Comment:
if you want to, you can rename the `headerList` to `headerMainList`, and then leave 
```
success: true,
headerMainList
```

it is called shorthand properties https://ui.dev/shorthand-properties

Reply:
Done